pkg/cloud-provider/cloudapi/aws/aws_ec2.go

```diff
@@ -268,8 +282,16 @@ func (ec2Cfg *ec2ServiceConfig) DoResourceInventory() error {
 			id := cloudcommon.InstanceID(strings.ToLower(aws.StringValue(instance.InstanceId)))
 			instanceIDs[id] = instance
 			vpcIDs[strings.ToLower(*instance.VpcId)] = exists
+			vpcIDTmp := make(map[string]struct{})
+			vpcIDTmp[strings.ToLower(*instance.VpcId)] = exists
+			networkInterfaces, err := ec2Cfg.getNetworkInterfacesOfVpc(vpcIDTmp)
+			if err != nil {
+				awsPluginLogger().Error(err, "failed to get network interfaces of vpcs", "vpc-ids", vpcIDTmp)
```

Comment:
There will be only one VPC at a time I believe, log the specific VPC ID, no need of providing the map.